app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+
+    alive = []
```

Comment:
The `alive` list is a class variable intended to track all living animals. Ensure that animals are properly removed from this list when they are no longer alive, as currently handled in the `bite` method.